paket.dependencies

```diff
@@ -2,17 +2,16 @@ framework: net45
 source https://www.nuget.org/api/v2
 
 nuget FSharp.Control.AsyncSeq
-nuget MongoDB.Bson ~> 2.0.0
-nuget MongoDB.Driver.Core ~> 2.0.0
+nuget MongoDB.Driver.Core
 
 // for building the project
 nuget FAKE
 nuget SourceLink.Fake
 
 // for executing tests
-nuget NUnit >= 2.6.2 < 3
-nuget NUnit.Runners >= 2.6.2 < 3
-nuget Unquote ~> 3
+nuget NUnit ~> 2
```

Comment:
The reason for the version constraints on NUnit, NUnit.Runners, and Unquote is because that's when those projects added support for .NET Framework 4.5. Perhaps to make this more obvious, it may be worth [setting `lowest_matching: true` in `paket.dependencies`](https://fsprojects.github.io/Paket/dependencies-file.html#Lowest_matching-option).